src/configure.py

```diff
@@ -33,8 +32,9 @@
 messages = {
     'hello': 'This application will help you configure TRD to manage payouts for your bakery. Type enter to continue',
     'bakingaddress': 'Specify your baking address public key hash (Processing may take a few seconds)',
-    'paymentaddress': 'Specify your payment PKH. Available addresses:{}',
+    'paymentaddress': 'Specify your payouts public key hash. It can be the same as your baking address, or a different one.',
```

Comment:
"Available addresses:" is removed. Could be useful for the user.

Reply:
It was removed because there was no code which was formatting the string for display. On my screen it literally showed "Available addresses:{}"